components/crud-web-apps/jupyter/frontend/src/app/app.component.ts

```diff
@@ -1,10 +1,16 @@
 import { Component } from '@angular/core';
+import { TranslateService } from '@ngx-translate/core';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.scss'],
 })
 export class AppComponent {
+  constructor(private translate: TranslateService) {
+    const currentLanguage = this.translate.getBrowserLang();
+    const lang = currentLanguage.match(/en|fr/) ? currentLanguage : 'en';
```

Comment:
Will new languages need to be added to the `currentLanguage.match(/en|fr/)` section once the JSON files are added?

Reply:
That is correct. Once a language's JSON file is added the corresponding `app.component.ts` will need to be updated as well